python_course_1604/class_01/exercize_06_while_break_else.py

```diff
@@ -37,9 +37,12 @@ def count_until(match_value):
     Please use while, break, else construct.
     """
     times = 0
-    while has_next():
-        val = get_next()
-        times += 1
-        if val == match_value:
-            return times
+    while True:
+        if has_next():
+            val = get_next()
+            times += 1
+            if val == match_value:
+                return times
+        else:
```

Comment:
Come on, that's not a break even if it breaks the flow. The instructions say "use while, break, else" :P